bench-vortex/src/bin/tpch_benchmark.rs

```diff
@@ -241,6 +253,7 @@ async fn bench_main(
             .zip_eq(EXPECTED_ROW_COUNTS)
             .enumerate()
             .filter(|(idx, _)| queries.as_ref().map(|q| q.contains(idx)).unwrap_or(true))
+            .filter(|(idx, _)| exclude_queries.as_ref().map(|excluded| !excluded.contains(idx)).unwrap_or(true))
```

Comment:
This has always been missing. If you exclude a query its row count is zero which fails the check unless the expected row count is zero.